src/stores/widgets/ElementWidgetActions.ts

```diff
@@ -37,6 +35,23 @@ export enum ElementWidgetActions {
      * @deprecated Use MSC2931 instead
      */
     ViewRoom = "io.element.view_room",
+    // This can be sent as from or to widget
```

Comment:
```suggestion
    // This can be sent from or to widget
```
Not sure, but this sounded a bit weird?